test/Feature/QuerySpec.hs

```diff
@@ -886,7 +886,7 @@ spec actualPgVersion = do
     it "only returns an empty result set if the in value is empty" $
       get "/items_with_different_col_types?int_data=in.( ,3,4)"
         `shouldRespondWith`
-        [json| {"hint":null,"details":null,"code":"22P02","message":"invalid input syntax for integer: \"\""} |]
+        [json| {"hint":null,"details":null,"code":"22P02","message":"invalid input syntax for type integer: \"\""} |]
```

Comment:
Ditto.

Reply:
adjusted accordingly, thanks again for pointing this out and for the detailed description on how to resolve this